examples/Header/Header.js

```diff
@@ -0,0 +1,35 @@
+import Component from '../Component';
```

Comment:
@todo does this make sense in this repo? Probably should move to a wiki

Reply:
I think we could leave this in here as an example component, just make sure it's not included in the package.json  `files` field

Reply:
I did it with the `.npmignore` file, I can add the files directive, too 👍